src/main/java/org/jenkinsci/plugins/DependencyCheck/aggregator/FindingsAggregator.java

```diff
@@ -29,17 +31,18 @@
 public class FindingsAggregator {
 
     private final SeverityDistribution severityDistribution;
-    private final List<Finding> aggregatedFindings;
+    private final SortedSet<Finding> aggregatedFindings;
     
     public FindingsAggregator(int buildNumber) {
         severityDistribution = new SeverityDistribution(buildNumber);
-        aggregatedFindings = new ArrayList<>();
+        aggregatedFindings = new TreeSet<>();
     }
 
     public void addFindings(List<Finding> findings) {
         for (Finding finding : findings) {
-            aggregatedFindings.add(finding);
-            severityDistribution.add(finding.getNormalizedSeverity());
+            if (aggregatedFindings.add(finding)) {
```

Comment:
since you had implement the hasCode and equals method is not needed to have a SortedSet. The list is sorted by the UI widget based on the column the user selection. To keep the information on how many project this dependency has impact (maybe useful to perform a weighted average chart in the future) add a counter attached to the findings.

Here you can change with
```
if (!aggregatedFindings.contains(finding)) {
    aggregatedFindings.add(finding);
    ...
} else {
    aggregatedFindings.increaseCount(1);
}
```